docs/self-managed/zeebe-deployment/configuration/broker.md

```diff
@@ -0,0 +1,576 @@
+| enableMonitoring   | Configure disk monitoring to prevent getting into a non-recoverable state due to out of disk space. When monitoring is enabled, the broker rejects commands and pause replication when the required freeSpace is not available. This setting can also be overridden using the environment variable `ZEEBE_BROKER_DATA_DISK_ENABLEMONITORING`                                                                                                                                                                                 | True          |
+| monitoringInterval | Sets the interval at which the disk usage is monitored. This setting can also be overridden using the environment variable `ZEEBE_BROKER_DATA_DISK_MONITORINGINTERVAL`                                                                                                                                                                                                                                                                                                                                                       | 1s            |
+| freeSpace          |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |               |
+| processing         | When the free space available is less than this value, this broker rejects all client commands and pause processing. This setting can also be overridden using the environment variable `ZEEBE_BROKER_DATA_DISK_FREESPACE_PROCESSING`                                                                                                                                                                                                                                                                                        | 2GB           |
+| replication        | When the free space available is less than this value, broker stops receiving replicated events. This value must be less than freeSpace.processing. It is recommended to configure free space large enough for at least one log segment and one snapshot. This is because a partition needs enough space to take a new snapshot to be able to compact the log segments to make disk space available again. This setting can also be overridden using the environment variable `ZEEBE_BROKER_DATA_DISK_FREESPACE_REPLICATION` | 1GB           |
```

Comment:
❓ these two are sub-properties of `freeSpace` would it make sense to indicate that in the table e.g. by naming them `freeSpace.processing` ? (other similar cases further down as well)

Reply:
Oh shoot! Good catch. The spacing was difficult to understand when removing the # from the template. I'll fix it.